integration/sawtooth_integration/tests/burrow_evm_intkey_test.go

```diff
@@ -89,8 +106,8 @@ func TestIntkey(t *testing.T) {
 
 }
 
-func exec(t *testing.T, client *burrow_evm_client.Client, priv, to, data []byte) {
-  _, err := client.Exec(priv, to, data, 1000)
+func exec(t *testing.T, c *client.Client, priv, to, data []byte, nonce uint64) {
```

Comment:
I will disagree with you that one-letter parameters are always bad. In this case it is a 3-line function in a test file, so I would argue that it is fine. If this was a much longer function with a lot going on, a more descriptive name would be needed.

To support my claim, I cite the Go code review comments page: https://github.com/golang/go/wiki/CodeReviewComments#variable-names